tutorials/export/exporting_for_android.rst

```diff
@@ -41,7 +41,7 @@ Download and install the Android SDK.
 
 ::
 
-    sdkmanager --sdk_root=<android_sdk_path> "platform-tools" "build-tools;30.0.3" "platforms;android-31" "cmdline-tools;latest" "cmake;3.10.2.4988404" "ndk;21.4.7075529"
+    sdkmanager --sdk_root=<android_sdk_path> "platform-tools" "build-tools;30.0.3" "platforms;android-34" "cmdline-tools;latest" "cmake;3.10.2.4988404" "ndk;21.4.7075529"
```

Comment:
Can you update the `build-tools` and the `ndk` versions as well, the `ndk` version is listed above but for some reasons it wasn't updated in this command.

Reply:
Sure!